x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_cell_actions.cy.ts

```diff
@@ -29,17 +29,17 @@ import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
 import { login } from '../../../tasks/login';
 import { visit } from '../../../tasks/navigation';
 import {
+  removeKqlFilter,
   fillAddFilterForm,
   fillKqlQueryBar,
   openAddFilterPopover,
-  removeKqlFilter,
 } from '../../../tasks/search_bar';
 import { openActiveTimeline } from '../../../tasks/timeline';
 
 import { ALERTS_URL } from '../../../urls/navigation';
 
 // Failing: See https://github.com/elastic/kibana/issues/193840
```

Comment:
Is there any reason why the comment has not been removed?

Reply:
just missed this one. Removing it in my next PR 😄

Reply:
Miswrote...my next commit* I have it on my local branch now

Reply:
Fixed. Now you can't prove it said next PR lol 😁

Reply:
haha, are you sure? 😝 
![Screenshot 2024-11-01 at 10 23 51 AM](https://github.com/user-attachments/assets/a8727ccb-6962-4ead-85e1-b36388f9dac1)

Reply:
fake news...lol